catroid/src/main/java/org/catrobat/catroid/ui/ScriptActivity.java

```diff
@@ -535,7 +528,9 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 			if (backStackEntryName != null
 					&& (backStackEntryName.equals(LookFragment.TAG) || backStackEntryName.equals(SoundFragment.TAG)
 					|| backStackEntryName.equals(BackPackScriptListFragment.TAG) || backStackEntryName.equals(BackPackLookListFragment
-					.TAG) || backStackEntryName.equals(BackPackSoundListFragment.TAG) || backStackEntryName.equals(NfcTagFragment.TAG))) {
+					.TAG) || backStackEntryName.equals(BackPackSoundListFragment.TAG) || backStackEntryName.equals(NfcTagFragment.TAG)
+					|| backStackEntryName.equals(BackPackUserBrickListFragment.TAG)
+					|| backStackEntryName.equals(FormulaEditorFragment.FORMULA_EDITOR_FRAGMENT_TAG))) {
```

Comment:
Very good idea, I've definitively refactored that (seems to me like a historically grown if-statement ;) )